### PR TITLE
table attribute completeness

### PR DESCRIPTION
**Pull request by morganen**

### CODE REVIEW

File: content/md/first-steps/manage-multidimensional-data-in-a-table.md [hunk 1]
```diff
@@ -138,6 +138,13 @@ That's why **we highly recommend you to keep the import/export feature for data
...
 
-As of today, we consider a table **complete**, as soon as **1 cell is filled**. Stay tuned, we will improve it soon.  
+The basic rule is that we consider a table **complete**, as soon as **1 cell is filled**. But you can go further by defining, in each column, if this one is required for the completeness of the attribute.
+
+![image](../img/TableAttributeCompleteness.png)
+
+This way, you can easily define that:
+- the table is complete if **every cells** are filled (you should have checked the completeness option in every column)
+- the table is complete if **every cells of 1 or several column.s** are filled (you should have checked the completeness option in some columns only)
+- the table is complete as soon as **one cell** is filled (including the cells of the first column).
```
Comment (nmarniesse) [inline content/md/first-steps/manage-multidimensional-data-in-a-table.md:148]:
```suggestion
- the table is complete as soon as **one cell** is filled (only the completeness option of the first column should be checked). This is the default.
```

**Review by nmarniesse — APPROVED** [review]

File: content/md/first-steps/manage-multidimensional-data-in-a-table.md [hunk 2]
```diff
@@ -138,6 +138,13 @@ That's why **we highly recommend you to keep the import/export feature for data
 
 You can define that a table is required for the completeness of a channel. See the [dedicated article](what-is-the-completeness.html).
 
-As of today, we consider a table **complete**, as soon as **1 cell is filled**. Stay tuned, we will improve it soon.  
+The basic rule is that we consider a table **complete**, as soon as **1 cell is filled**. But you can go further by defining, in each column, if this one is required for the completeness of the attribute.
+
+![image](../img/TableAttributeCompleteness.png)
+
+This way, you can easily define that:
+- the table is complete if **every cells** are filled (you should have checked the completeness option in every column)
```
Comment (mmetayer) [inline content/md/first-steps/manage-multidimensional-data-in-a-table.md:146]:
```suggestion
- the table is complete if **every cell** is filled (you should have checked the completeness option in every column)
```

File: content/md/first-steps/manage-multidimensional-data-in-a-table.md [hunk 3]
```diff
@@ -138,6 +138,13 @@ That's why **we highly recommend you to keep the import/export feature for data
...
+- the table is complete if **every cells of 1 or several column.s** are filled (you should have checked the completeness option in some columns only)
```
Comment (mmetayer) [inline content/md/first-steps/manage-multidimensional-data-in-a-table.md:147]:
```suggestion
- the table is complete if **every cell of 1 or several column.s** is filled (you should have checked the completeness option in some columns only)
```

**Review by mmetayer — APPROVED** [review]

File: content/md/first-steps/manage-multidimensional-data-in-a-table.md [hunk 4]
```diff
@@ -138,6 +138,13 @@ That's why **we highly recommend you to keep the import/export feature for data
 
 You can define that a table is required for the completeness of a channel. See the [dedicated article](what-is-the-completeness.html).
 
-As of today, we consider a table **complete**, as soon as **1 cell is filled**. Stay tuned, we will improve it soon.  
+The basic rule is that we consider a table **complete**, as soon as **1 cell is filled**. But you can go further by defining, in each column, if this one is required for the completeness of the attribute.
```
Comment (camillebourjade) [inline content/md/first-steps/manage-multidimensional-data-in-a-table.md:141]:
```suggestion
The ground rule is that we consider a table **complete** as soon as **one cell is filled**. You can go further by defining the completeness at the column level by making it required for the completeness of the attribute.
```

File: content/md/first-steps/manage-multidimensional-data-in-a-table.md [hunk 5]
```diff
@@ -138,6 +138,13 @@ That's why **we highly recommend you to keep the import/export feature for data
...
+
+![image](../img/TableAttributeCompleteness.png)
+
+This way, you can easily define that:
```
Comment (camillebourjade) [inline content/md/first-steps/manage-multidimensional-data-in-a-table.md:145]:
```suggestion
This way, you can easily define three different levels of completeness:
```

File: content/md/first-steps/manage-multidimensional-data-in-a-table.md (hunk 3 above)
Comment (camillebourjade) [reply content/md/first-steps/manage-multidimensional-data-in-a-table.md:147]:
```suggestion
- the table is complete if **every cell of at least one column** is filled (you should have checked the completeness option for one or several columns)
```

File: content/md/first-steps/manage-multidimensional-data-in-a-table.md [hunk 6]
```diff
@@ -138,6 +138,13 @@ That's why **we highly recommend you to keep the import/export feature for data
...
+- the table is complete if **every cells** are filled (you should have checked the completeness option in every column)
+- the table is complete if **every cells of 1 or several column.s** are filled (you should have checked the completeness option in some columns only)
+- the table is complete as soon as **one cell** is filled (only the completeness option of the first column should be checked). This is the default.
```
Comment (camillebourjade) [inline content/md/first-steps/manage-multidimensional-data-in-a-table.md:148]:
```suggestion
- the table is complete as soon as **one cell** is filled. In that case, only the completeness option of the first column should be checked. This is the default behavior.
```

**Review by camillebourjade — APPROVED** [review]
Engligh Review

**Pull request merged**